css-custom-functions/responsive-narrow-wide/style.css

```
@@ -0,0 +1,29 @@
+@function --narrow-wide(--narrow, --wide) {
+  result: var(--wide);
+  @media (width < 700px) {
+    result: var(--narrow);
+  }
+}
+
+/*
+This could also be written using a CSS if() function, like so:
+@function --narrow-wide(--narrow, --wide) {
+  result: if(media(width < 700px): var(--narrow) ; else: var(--wide));
+}
+*/
+
+body {
+  display: grid;
+  grid-template-columns: repeat(--narrow-wide(1, 3), 1fr);
+  gap: --narrow-wide(0, 20px);
+  padding: 0 20px;
```

```suggestion
  padding: 0 20px;
  font-family: system-ui;
```

Unlike the other demos, you haven’t used the `Helvetica, Arial, sans-serif` spell here. Have you considered using this modern generic font-family value? It’s been Baseline widely available for a while now. It would help the demo look “styled”, you know.

---

This is a good tip; I've added it to all three demos in my next commit.